control/action.yaml

```diff
@@ -0,0 +1,44 @@
+openapi: 3.1.0
+
+info:
+  title: Action models
+  description: >-
+    All Detail Action schemas
```

Comment:
@ajbalogh @PrasenjitAdhikary probably needs a more elaborate description here, it's not clear what/why/how it is applied.

Reply:
The info block is redundant and not required, there is only one info block object which is found in the top level info.yaml file.